src/main/java/io/aiven/kafka/auth/json/AivenAcl.java

```diff
@@ -38,26 +38,34 @@ public class AivenAcl {
     @SerializedName("resource_pattern")
     public final String resourceRePattern;
 
+    @SerializedName("permission_type")
+    private final AclPermissionType permissionType;
+
     public AivenAcl(final String principalType,
                     final String principal,
                     final String operation,
                     final String resource,
-                    final String resourcePattern) {
+                    final String resourcePattern,
+                    final AclPermissionType permissionType) {
         this.principalType = principalType;
         this.principalRe = Pattern.compile(principal);
         this.operationRe = Pattern.compile(operation);
-        this.resourceRe = Objects.nonNull(resource)
-            ? Pattern.compile(resource) : null;
+        this.resourceRe = Objects.nonNull(resource) ? Pattern.compile(resource) : null;
         this.resourceRePattern = resourcePattern;
+        this.permissionType = Objects.isNull(permissionType) ? AclPermissionType.ALLOW : permissionType;
```

Comment:
I would suggest using the "boring" form :) `Objects.isNull` was meant to be used as a predicate for filters and the like, not to replace the normal comparison
```suggestion
        this.permissionType = permissionType == null ? AclPermissionType.ALLOW : permissionType;
```